### PR TITLE
Fix code smells

### PR DESCRIPTION
**Pull request by sdimitrov9**
**Description**:
200+ sonar issues are expected to be fixed with this PR

**Related issue(s)**:

Partially fixes 9010

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-mirror-grpc/src/test/java/com/hedera/mirror/grpc/repository/TopicMessageRepositoryTest.java [hunk 1]
```diff
@@ -48,8 +48,6 @@ void findByFilterEmpty() {
     @Test
     void findByFilterNoMatch() {
         TopicMessage topicMessage1 = domainBuilder.topicMessage().block();
-        TopicMessage topicMessage2 = domainBuilder.topicMessage().block();
-        TopicMessage topicMessage3 = domainBuilder.topicMessage().block();
```
Comment (steven-sheehy) [inline hedera-mirror-grpc/src/test/java/com/hedera/mirror/grpc/repository/TopicMessageRepositoryTest.java:52]: Even though the variable is not referenced, these topic messages need to be created in the database to properly test the scenario. Maybe you can just remove the field declarations. 

Same comment for other changes in this file.

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: hedera-mirror-grpc/src/test/java/com/hedera/mirror/grpc/repository/TopicMessageRepositoryTest.java (hunk 1 above)
Comment (sdimitrov9) [reply hedera-mirror-grpc/src/test/java/com/hedera/mirror/grpc/repository/TopicMessageRepositoryTest.java:52]: Fixed.

**Review by steven-sheehy — DISMISSED** [review]
LGTM

**Comment by steven-sheehy** [comment]
Looks like your change violates spotless check

**Comment by sdimitrov9** [comment]
> Looks like your change violates spotless check

Fixed + additional fixes added to the latest commit

**Review by steven-sheehy — APPROVED** [review]

**Pull request merged**